paparazzi/paparazzi-gradle-plugin/src/main/java/app/cash/paparazzi/gradle/PrepareResourcesTask.kt

```diff
@@ -49,6 +49,10 @@ abstract class PrepareResourcesTask : DefaultTask() {
   @get:PathSensitive(PathSensitivity.RELATIVE)
   abstract val mergeAssetsOutput: DirectoryProperty
 
+  @get:InputDirectory
+  @get:PathSensitive(PathSensitivity.ABSOLUTE)
+  abstract val snapshotDirectory: DirectoryProperty
```

Comment:
I'm really confused about naming
```
task.snapshotDirectory.set(snapshotOutputDir)
```
is it `@InputDirectory` or `outputDir`?

should the extension be just `snapshotDir(ectory)`?

Reply:
I have a feeling this would become either
```suggestion
  @get:Input
  abstract val snapshotDirectory: Property<String>
```
or
```suggestion
  @get:InputDirectory
  @get:Optional
  @get:PathSensitive(PathSensitivity.RELATIVE)
  abstract val snapshotDirectory: DirectoryProperty
```
and do a similar relativize as the other props.

Reply:
That said `@get:InputDirectory` still seems wrong, because the contents of the folder don't matter, just the folder location.

Reply:
Will this task actually write the contents of that directory?

Reply:
Should this task re-execute if a screenshot changes inside the directory or stay up to date?

Reply:
resolve